x-pack/plugins/alerting/server/alerts_client/alerts_client.ts

```diff
@@ -413,7 +413,7 @@ export class AlertsClient<
       this.legacyAlertsClient.getAlertsToSerialize(false);
 
     const activeAlerts = this.legacyAlertsClient.getProcessedAlerts('active');
-    const recoveredAlerts = this.legacyAlertsClient.getProcessedAlerts('recovered');
+    const currentRecoveredAlerts = this.legacyAlertsClient.getProcessedAlerts('recoveredCurrent');
 
```

Comment:
`currentRecoveredAlerts` are used to determine if the alert is from the current execution or previous one.
But `activeAlerts` are just used to get legacyAlerts.